EDDiscovery/EDDiscovery.csproj

```diff
@@ -848,40 +833,37 @@
     </Task>
   </UsingTask>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <PropertyGroup>
-    <PostBuildEvent>
-    </PostBuildEvent>
-  </PropertyGroup>
   <!-- To modify your build process, add your task inside one of the targets below and uncomment it.
        Other similar extension points exist, see Microsoft.Common.targets.
   <Target Name="BeforeBuild">
   </Target>
   -->
-  <Target Name="AfterBuild" Condition=" '$(OS)' != 'Unix' ">
+  <Target Name="AfterBuild" Outputs="$(OutDir)EDDiscovery.Portable.zip" Condition="'$(OS)' != 'Unix'">
     <ItemGroup>
-      <ZipFiles Include="$(OutputPath)\EDDiscovery.exe" />
-      <ZipFiles Include="$(OutputPath)\EDDiscovery.pdb" />
-      <ZipFiles Include="$(OutputPath)\SharpDX.DirectInput.dll" />
-      <ZipFiles Include="$(OutputPath)\SharpDX.dll" />
-      <ZipFiles Include="$(ProjectDir)\App.Portable.config">
+      <ZipFiles Include="$(TargetPath)" />
+      <ZipFiles Include="$(OutDir)*.dll" />
+      <ZipFiles Include="$(OutDir)*.pdb" />
+      <ZipFiles Include="$(ProjectDir)App.Portable.config">
         <Name>EDDiscovery.exe.config</Name>
       </ZipFiles>
-      <ZipFiles Include="$(OutputPath)\CSCore.dll" />
-      <ZipFiles Include="$(OutputPath)\Newtonsoft.Json.dll" />
-      <ZipFiles Include="$(OutputPath)\OpenTK.GLControl.dll" />
-      <ZipFiles Include="$(OutputPath)\OpenTK.dll" />
-      <ZipFiles Include="$(OutputPath)\ActionLanguage.dll" />
-      <ZipFiles Include="$(OutputPath)\Audio.dll" />
-      <ZipFiles Include="$(OutputPath)\Conditions.dll" />
-      <ZipFiles Include="$(OutputPath)\DirectInput.dll" />
-      <ZipFiles Include="$(OutputPath)\ExtendedControls.dll" />
-      <ZipFiles Include="$(OutputPath)\General.dll" />
+      <ZipFiles Include="$(OutDir)System.Data.SQLite.xml" />
+      <ZipFiles Include="@(SQLiteInteropFiles)">
+        <Name>%(RecursiveDir)%(Filename)%(Extension)</Name>
+      </ZipFiles>
     </ItemGroup>
-    <Zip OutputFileName="$(OutputPath)\EDDiscovery.Portable.zip" Files="@(ZipFiles)" />
+    <Message Text="BuildPortableZip:" />
+    <Message Text="'@(ZipFiles)' -> '$(OutDir)EDDiscovery.Portable.zip'" />
+    <Zip OutputFileName="$(OutDir)EDDiscovery.Portable.zip" Files="@(ZipFiles)" />
+    <Message Condition="!Exists('$(OutDir)EDDiscovery.Portable.zip')" Importance="high" Text="Unknown error in BuildPortableZip." />
```

Comment:
Correction (can't edit?) The first message can go away, the third should be an error.